ivy/functional/frontends/numpy/statistics/averages_and_variances.py

```diff
@@ -11,6 +11,25 @@
 )
 
 
+def nanmedian(a, /, *, axis=None, keepdims=False, out=None):
+    is_nan = ivy.isnan(a)
+    axis = tuple(axis) if isinstance(axis, list) else axis
+
+    if not ivy.any(is_nan):
+        ret = ivy.median(
+            a, axis=axis, keepdims=keepdims, out=out
+        )
+    else:
+        a = [i for i in a if ivy.isnan(i) is False]
+        ret = ivy.median(
+            a, axis=axis, keepdims=keepdims, out=out
+        )
+    return ret
```

Comment:
please change this line to "  return ret[0]" as this is the function that is being tested.